llvm/lib/Target/EraVM/EraVMBytesToCells.cpp

```diff
@@ -60,22 +61,6 @@ class EraVMBytesToCells : public MachineFunctionPass {
   /// true if a conversion has been done.
   void convertCodeAddressMachineInstr(MachineInstr::mop_iterator OpIt);
 
-  /// return iterator to the stack access operand of \p MI. If there is no stack
```

Comment:
Can we avoid moving this code to the `EraVMInstrInfo.cpp`, and use helpers like `argumentType` to achieve the same thing?